manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SERPRO.AssinadorSERPRO
+PackageVersion: 4.0.2
+PackageLocale: pt-BR
+Publisher: SERPRO
+PackageName: Assinador SERPRO
```

```suggestion
PackageName: Assinador Serpro
```
Should match control panel